### ISSUE TITLE
Create Docker recipe for logs

### ISSUE THREAD

**Issue by rishimukhop1** [newrelic/open-install-library#95]
The Docker recipe does not need to install any specific component, as Docker monitoring is available OOB through the infra agent.

The only thing we need to include in this recipe are the common log files that Docker writes. This recipe needs to be served up by the recipe service when Docker processes are detected on the system.

Default log files are located in{{ /var/lib/docker/containers/CONTAINER\_ID/CONTAINER\_ID-json.log.}} We need to develop regex to pick these up across all the containers. Note that the log files will be populated only based on whatever containers are in that directory at the time of install. Adding additional log files based on new containers that are spawned are out of scope at this time.

Aha! Link: https://newrelic.aha.io/features/VIRTUOSO-97

**Comment by rishimukhop1** [comment]
Comment added by Justin Eveland in Aha! - [View](https://newrelic.aha.io/comments/6921827982345929323)

Ah, let me clarify something. The way it was implemented in the current PR is such that the user has no indication that we're collecting Docker log files. That's because the setup is done in the existing `logs.yml` file, and does a check to see if `/var/log/docker/containers` exists, and if it does it starts collecting logs.  
  
  
The case I'm making is: we ask for confirmation that we're going to collect log files for everything else, and I believe Docker should be no different. It'd go against the patterns we've developed to put that logic in the existing `logs.yml` because we do IO control prior to the execution of go-task. So we should just use a separate `docker-logs.yml` and utilize process detection so the CLI can pull that recipe and set up collection of Docker logs.
